### PR TITLE
Doppel Vision v1.0.0

### PR DESCRIPTION
**Pull request by saipriya-metron**
# PR Details

This is a new PR for Sumologic - Doppel Integration

## Description

Doppel Vision manages external threats at the speed of AI. Doppel technology identifies and takes down deep fakes, malicious impersonations, phishing, and disinformation campaigns targeting clients, and utilizes proprietary AI and machine learning tools to automate threat detection and takedowns.
These dynamic features enable Doppel Vision to scale with you as your partner in brand protection in an evolving threat landscape.

## Checklist

- [ x ] Changes follows the best practices from the given [docs](https://docs.google.com/document/d/1xfYfruru0RFWOH23GRrRzosSlepJUTcakLFHLHdpEtc/edit?usp=sharing).
- [ x ] Ran unit tests locally.
- [ x ] Updated screenshots/sample logs if any.

### CODE REVIEW

**Comment by himanshu219** [comment]
1.. We generally do not create log searches for all the panels part of dashboards
2. Can we add use cases like recent critical alerts, top 10 products with critical alerts, Alerts by Severity (pie chart),

**Comment by saipriya-metron** [comment]
> 1.. We generally do not create log searches for all the panels part of dashboards 2. Can we add use cases like recent critical alerts, top 10 products with critical alerts, Alerts by Severity (pie chart),

Our client Doppel prefers the dashboard to closely resemble the Zerofox dashboard. As such, we followed a similar approach, utilising log searches for each panel in line with the client's expectations.

File: DoppelVision/DoppelVision.manifest.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+    "author": "DoppelVision",
+    "categories": [
+        "Compliance and Security",
+        "Sumo Logic Certified"
+    ],
+    "communityURL": "https://support.sumologic.com/hc/en-us/community/topics/200263058-Applications-and-Integrations",
+    "description": "Doppel Vision manages external threats at the speed of AI. Doppel technology identifies and takes down deep fakes, malicious impersonations, phishing, and disinformation campaigns targeting clients, and utilizes proprietary AI and machine learning tools to automate threat detection and takedowns.\nThese dynamic features enable Doppel Vision to scale with you as your partner in brand protection in an evolving threat landscape.",
+    "helpURL": "https://docs.google.com/document/d/1XA3KB1QkhQudkVsNqclyyNOvSipJJGujyyplaKL0Fz8/edit",
```
[the remark this answers is not in the document]
Comment (saipriya-metron) [reply DoppelVision/DoppelVision.manifest.json:9]: We can update the helpURL to reference the README file. All the necessary steps are already documented in the README.

**Comment by himanshu219** [comment]
Also lets add filter variables for alert_id, severity, entity_state and product so that users can quickly filter these alerts.
Also we do not have to remove panels, you can just adjust their width and height and add above use cases. Here's the json where I added the two panels.

image

[Doppel_Vision_Dashboard (1).json](https://github.com/user-attachments/files/16976606/Doppel_Vision_Dashboard.1.json)

**Comment by saipriya-metron** [comment]
> Also lets add filter [variables](https://help.sumologic.com/docs/dashboards/filter-template-variables/) for alert_id, severity, entity_state and product so that users can quickly filter these alerts.
> 
> Also we do not have to remove panels, you can just adjust their width and height and add above use cases. Here's the json where I added the two panels.
> 
> image
> [Doppel_Vision_Dashboard (1).json](https://github.com/user-attachments/files/16976606/Doppel_Vision_Dashboard.1.json)

Thank you for your feedback. Regarding the UI changes, we are unable to modify the layout or structure of the dashboard as it adheres to the approved version from our client, Doppel. However, we are implementing filters for the three fields currently in use on the dashboard: alert_id, product, and queue_state. Once we have completed our testing, we will update the PR accordingly.

We appreciate your understanding.

**Comment by himanshu219** [comment]
LGTM

**Pull request merged**